docs/components/configurationPanel.md

```diff
@@ -0,0 +1,224 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.configurationPanel`
+
+This is a `viewComponent` and handles the state and functionality of the configuration panel. It also creates the
```

Comment:
I'd prefer the full gradeName, i.e. `fluid.viewComponent`. Please link to the docs as well.

Reply:
"state and functionality" is not very clear, nor is "creates the configuration options".  I'd suggest reworking this whole paragraph.